ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/app/server.clj

```diff
@@ -79,7 +80,8 @@
      (-> (create-ring-handler)
          (httpkit/start-container port))
      (-> (create-ring-handler)
-         (aleph/start-container port)))))
+         (aleph/start-container port)))
+   (graphite/start-graphite-reporter)))
```

Comment:
Good catch.  I've moved the metrics process before the starting of the containers but after the definition of the ring handler, where the metrics are defined.  The start/stop should work correctly from the REPL now.  The previous implementation was working from the `init.d` script, but only because the stop function isn't used there.